src/components/Home.jsx

```diff
@@ -31,11 +32,18 @@ class Home extends Component {
 
   render() {
     const { wrapper } = this.props
+    // FIXME
+    const hasCustomBG = false
```

Comment:
This is temporary, until custom background is implemented. The style is done though. Just this part is to be removed when time comes.

Reply:
Sure. I honestly didn't know what was the best way to add some code not really usable at this point. I'm fine with your way. :)